### PR TITLE
Bump dependency versions (checkers; Gradle)

### PR DESCRIPTION
**Pull request by stkent**
See commit comments for details.

FindBugs has no update available; will open an issue to swap to SpotBugs.

### CODE REVIEW

File: example-java/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```
Comment (stkent) [inline example-java/gradle/wrapper/gradle-wrapper.properties]: Should remove these files, this example dir was left in my local copy of the repo but has been removed from upstream.

Comment (btkelly) [reply example-java/gradle/wrapper/gradle-wrapper.properties]: So wait, did you remove these? Or are you saying that should be in a different PR?

Comment (stkent) [reply example-java/gradle/wrapper/gradle-wrapper.properties]: About to delete them and push again.

**Comment by stkent** [comment]
@btkelly updated!

**Pull request merged**